nntrainer/include/optimizer.h

```diff
@@ -119,27 +95,19 @@ class Optimizer {
    * @brief     get Learning Rate
    * @retval    Learning rate
    */
-  float getLearningRate() { return popt.learning_rate; };
+  float getLearningRate() { return learning_rate; };
```

Comment:
`virtual float getLearningRate()` and change `adam::getLearningRate` to float?

Reply:
I think you are mixing two things.
There are two functionsin optimizer.h:
`float getLearningRate()` which is a public function
`virtual double getLearningRate(int iteration)` which is a private function and overloaded in adam.h

Reply:
Got it